SU-SWS/stanford_capx
# DEVOPS-000: Update composer-managed dependencies

# READY FOR REVIEW

# Summary
- Dependency updates

# Needed By (Date)
- Whenever

# Criticality
- How critical is this PR on a 1-10 scale? 2/10
- See https://github.com/SU-SWS/acsf-leland/pull/106

# Steps to Test

1. Check out this branch
2. Connect to CAP
3. Import profiles

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
